### PR TITLE
Kim edn

### PR DESCRIPTION
**Pull request by yafshar**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml" 
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by yafshar** [comment]
For the appveyor failure, it seems to be unrelated. Can anyone help?

File: recipes/kim_edn/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: bab88a0a890598577a5dabe0ff19ed573ef7a0be547e5a47b15deeb4ec711db5
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  skip: True  # [py<36]
```
Comment (npavlovikj) [inline recipes/kim_edn/meta.yaml]: I think this recipe can be `noarch`, https://conda-forge.readthedocs.io/en/latest/meta.html#building-noarch-packages. In that case, please remove the `skip` line, and add `noarch: python` instead.

File: recipes/kim_edn/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+requirements:
+  host:
+    - python
```
Comment (npavlovikj) [inline recipes/kim_edn/meta.yaml]: With `noarch`, please use `python >=3.6`.

File: recipes/kim_edn/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+    - pip
+  run:
+    - python
```
Comment (npavlovikj) [inline recipes/kim_edn/meta.yaml]: With `noarch`, please use `python >=3.6`.

File: recipes/kim_edn/meta.yaml (hunk 2 above)
Comment (yafshar) [reply recipes/kim_edn/meta.yaml]: @npavlovikj  I did noarch at first, but it complained that I can not use selector with noarch

File: recipes/kim_edn/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+  imports:
+    - kim_edn
+
+about:
+  home: https://github.com/yafshar/kim_edn
+  license: CDDL-1.0
+  license_family: OTHER
+  license_file: LICENSE.CDDL
+  summary: 'KIM-EDN encoder and decoder for Python'
+
```
Comment (npavlovikj) [inline recipes/kim_edn/meta.yaml]: Please remove empty line.

File: recipes/kim_edn/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+  # The remaining entries in this section are optional, but recommended.
```
Comment (npavlovikj) [inline recipes/kim_edn/meta.yaml]: Please remove comment.

File: recipes/kim_edn/meta.yaml (hunk 2 above)
Comment (npavlovikj) [reply recipes/kim_edn/meta.yaml]: Right, you can not have `noarch` and `skip` together. Please remove `skip`, and follow the other suggestions.

File: recipes/kim_edn/meta.yaml [hunk 6]
```diff
@@ -10,9 +10,9 @@ source:
   sha256: bab88a0a890598577a5dabe0ff19ed573ef7a0be547e5a47b15deeb4ec711db5
 
 build:
+  noarch: python >=3.6
```
Comment (npavlovikj) [inline recipes/kim_edn/meta.yaml]: Just `noarch: python` please.

Comment (yafshar) [reply recipes/kim_edn/meta.yaml]: Again the appveyor fails, but if you look at the failure message it does not have anything to do with my package.

File: recipes/kim_edn/meta.yaml [hunk 7]
```diff
@@ -0,0 +1,46 @@
...
+  sha256: bab88a0a890598577a5dabe0ff19ed573ef7a0be547e5a47b15deeb4ec711db5
+
+build:
+  noarch: python >=3.6
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python
```
Comment (npavlovikj) [inline recipes/kim_edn/meta.yaml]: Use `python >=3.6` in the `host` and `run` sections only.

File: recipes/kim_edn/meta.yaml (hunk 6 above)
Comment (npavlovikj) [reply recipes/kim_edn/meta.yaml]: We are just cleaning the recipe now and making it according to the conda-forge standards. `noarch` recipes are built only on Linux and save lots of CI time. 

The AppVeyor errors are common and not related to your package. For `noarch` recipes we just want the Linux builds to pass, for non `noarch` recipes, only one Windows build is ok, so we can ignore the AppVeyor errors.

**Comment by npavlovikj** [comment]
The recipe LGTM now, so once the tests finish, I can go ahead and merge it.

**Comment by yafshar** [comment]
> The recipe LGTM now, so once the tests finish, I can go ahead and merge it.
@npavlovikj thanks for the help.

**Comment by npavlovikj** [comment]
Thank you for contributing @yafshar 🎉

**Pull request merged**